File: src/main/java/com/gildedrose/java/Sulfuras.java
package com.gildedrose.java;

public class Sulfuras extends Item {
    public Sulfuras(String name, int sellIn, int quality) {
        super(name, sellIn, quality);
    }

    @Override
    protected void updateQuality() {
        // Sulfuras does not change in quality
    }

    @Override
    protected void decreaseSellIn() {
        // Sulfuras does not change sellIn
    }
}

File: src/main/kotlin/com/gildedrose/kotlin/Sulfuras.kt
package com.gildedrose.kotlin

class Sulfuras(name: String, sellIn: Int, quality: Int) : Item(name, sellIn, quality) {
    override fun updateQuality() {
        // Sulfuras does not change in quality
    }

    override fun decreaseSellIn() {
        // Sulfuras does not change sellIn
    }
}